api/handler/acl.go

```diff
@@ -34,9 +34,10 @@ var actionToOpMap = map[string][]eacl.Operation{
 }
 
 const (
-	arnAwsPrefix     = "arn:aws:s3:::"
-	allUsersWildcard = "*"
-	allUsersGroup    = "http://acs.amazonaws.com/groups/global/AllUsers"
+	arnAwsPrefix      = "arn:aws:s3:::"
+	allUsersWildcard  = "*"
+	allUsersGroup     = "http://acs.amazonaws.com/groups/global/AllUsers"
+	attributeObjectID = "$Object:objectID"
```

Comment:
You can use [const](https://github.com/nspcc-dev/neofs-api-go/blob/6d0b3135ff2337be338a8c5cef5449ac3a06b58e/v2/acl/filters.go#L11) from api-go, even though it is major version dependent.

Reply:
Fixed